Bypass CSP for webkit-pdfjs-viewer by default

TingPing opened the pull request:
#### 374d0b0b4520a44242ee475819810f438e39507d
<pre>
Bypass CSP for webkit-pdfjs-viewer by default
<a href="https://bugs.webkit.org/show_bug.cgi?id=243324">https://bugs.webkit.org/show_bug.cgi?id=243324</a>

Reviewed by NOBODY (OOPS!).

WebKit bundles PDF.js under the webkit-pdfjs-viewer scheme which could
get blocked if a website uses CSP.

* Source/WebCore/platform/LegacySchemeRegistry.cpp:
(WebCore::builtinCSPBypassingSchemes):
(WebCore::WTF_REQUIRES_LOCK):
</pre>

mcatanzaro reviewed (approved):
Thanks. I expect this should prevent CSP from accidentally blocking access to PDF.js content, but not allow PDF.js to access arbitrary content.

TingPing committed:
Bypass CSP for webkit-pdfjs-viewer by default
https://bugs.webkit.org/show_bug.cgi?id=243324

Reviewed by Michael Catanzaro.

WebKit bundles PDF.js under the webkit-pdfjs-viewer scheme which could
get blocked if a website uses CSP.

* Source/WebCore/platform/LegacySchemeRegistry.cpp:
(WebCore::builtinCSPBypassingSchemes):
(WebCore::WTF_REQUIRES_LOCK):

Canonical link: https://commits.webkit.org/253040@main
